No member named ‘emplace_back’

Getting the following error while running make:

RFSCTraceBuilder.cpp:947:15: error: ‘RFSCTraceBuilder::UnfoldingNodeChildren {aka class llvm::SmallVector<std::weak_ptr<RFSCTraceBuilder::UnfoldingNode>, 1u>}’ has no member named ‘emplace_back’; did you mean ‘push_back’?
   parent_list.emplace_back(c);
               ^~~~~~~~~~~~
Can anyone suggest what could be the possible reason?
[Already using gcc-6.5]

--- a/README
+++ b/README
@@ -130,12 +130,12 @@ Specifying Compiler
    When the configure script is invoked, it will by GNU autotools
    magic determine which C++ compiler will be used during
    compilation. In case e.g. your default compiler does not support
-   C++11, but you have the compiler g++-4.6 installed at a
+   C++14, but you have the compiler g++-5 installed at a
    non-standard location you may want to override this. In order to do
-   so, specify the path to g++-4.6 in CXX when invoking the
+   so, specify the path to g++-5 in CXX when invoking the
    './configure' command:
 
-   $ ./configure CXX='/path/to/g++-4.6'
+   $ ./configure CXX='/path/to/g++-5'
 
 Specifying LLVM Installation
 ----------------------------

--- a/src/RFSCTraceBuilder.cpp
+++ b/src/RFSCTraceBuilder.cpp
@@ -944,7 +944,7 @@ find_unfolding_node(UnfoldingNodeChildren &parent_list,
   /* Did not exist, create it. */
   std::shared_ptr<UnfoldingNode> c =
     std::make_shared<UnfoldingNode>(parent, read_from);
-  parent_list.emplace_back(c);
+  parent_list.push_back(c);
   return c;
 }
 

